src/request/create/IdenticonSelector.css

```diff
@@ -52,13 +54,27 @@
     position: relative;
 }
 
-.identicon-selector .wrapper:nth-child(1) { transform: translate(  0.0rem, -14.25rem); }
-.identicon-selector .wrapper:nth-child(2) { transform: translate(-12.5rem,  -7.25rem); }
-.identicon-selector .wrapper:nth-child(3) { transform: translate( 12.5rem,  -7.25rem); }
-.identicon-selector .wrapper:nth-child(4) { transform: translate(  0.0rem,   0.00rem); }
-.identicon-selector .wrapper:nth-child(5) { transform: translate(-12.5rem,   7.25rem); }
-.identicon-selector .wrapper:nth-child(6) { transform: translate( 12.5rem,   7.25rem); }
-.identicon-selector .wrapper:nth-child(7) { transform: translate(  0.0rem,  14.25rem); }
+.identicon-selector .wrapper:nth-child(1) { transform: translate(  0.0rem, -14.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(1):focus,
+.identicon-selector .wrapper:nth-child(1):hover { transform: translate(  0.0rem, -14.25rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(2) { transform: translate(-12.5rem,  -7.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(2):focus,
+.identicon-selector .wrapper:nth-child(2):hover { transform: translate(-12.5rem,  -7.25rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(3) { transform: translate( 12.5rem,  -7.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(3):focus,
+.identicon-selector .wrapper:nth-child(3):hover { transform: translate( 12.5rem,  -7.25rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(4) { transform: translate(  0.0rem,   0.00rem) scale(1); }
+.identicon-selector .wrapper:nth-child(4):focus,
+.identicon-selector .wrapper:nth-child(4):hover { transform: translate(  0.0rem,   0.00rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(5) { transform: translate(-12.5rem,   7.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(5):focus,
+.identicon-selector .wrapper:nth-child(5):hover { transform: translate(-12.5rem,   7.25rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(6) { transform: translate( 12.5rem,   7.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(6):focus,
+.identicon-selector .wrapper:nth-child(6):hover { transform: translate( 12.5rem,   7.25rem) scale(1.1); }
+.identicon-selector .wrapper:nth-child(7) { transform: translate(  0.0rem,  14.25rem) scale(1); }
+.identicon-selector .wrapper:nth-child(7):focus,
+.identicon-selector .wrapper:nth-child(7):hover { transform: translate(  0.0rem,  14.25rem) scale(1.1); }
```

Comment:
I think what you can do here to make this look nicer and remove code duplication, is translate-transform the `.wrapper`, but scale-transfrom the `.wrapper > .identicon`.

Reply:
Should be possible. For this, you'd need to set the default identicon scale to 1 and the `pop-in` `animation-direction` to `backwards`.

Reply:
`.wrapper` translated to correct position
`.wrapper .identicon` for the pop-in scale
`.wrapper .identicon img` for `:hover` and `:active` scale